CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -297,10 +291,16 @@ public function upgrade_4_7_15($rev) {
    * @param string $rev
    */
   public function upgrade_4_7_16($rev) {
-    $this->addTask(ts('Upgrade DB to %1: SQL', array(1 => $rev)), 'runSql', $rev);
+    $this->addTask('CRM-19723 - Add icon column to civicrm_option_value', 'addColumn',
+      'civicrm_option_value', 'icon', "varchar(255) COMMENT 'crm-i icon class' DEFAULT NULL");
+    $this->addTask('CRM-19769 - Add color column to civicrm_tag', 'addColumn',
+      'civicrm_tag', 'color', "varchar(255) COMMENT 'Hex color value e.g. #ffffff' DEFAULT NULL");
```

Comment:
@colemanw interesting, my mental model of `varchar(nnn)` was a bit wrong!

FWIW... one of those answers cites http://stackoverflow.com/a/1151833/4195300, but it seems to give a more complicated view -- i.e. the length shouldn't affect disk usage but may affect query performance.